src/test/groovy/org/prebid/server/functional/util/privacy/TcfConsent.groovy

```diff
@@ -102,4 +101,21 @@ class TcfConsent implements ConsentString {
             this.value = value
         }
     }
+
+    enum TcfPolicyVersion {
+
+        TCF_V2(2),
+        TCF_V3(4),
+        TCF_INVALID(63)
```

Comment:
Why not -1?

Reply:
I initially considered utilizing negative values as well, but I encountered a problem with the `TCStringEncoder` in the `iabtcf library`. While encoding this for tests, it consistently uses the maximum positive number due to its internal logic inside of `checkBounds` method - `(value & ((1L << Integer.SIZE) - 1), field);`. This behavior arises from how it masks the value to ensure it is within the valid range for an integer.

Modifying this would necessitate altering the logic of the iabtcf library, which is not ideal without careful consideration of potential side effects, especially given the library's wide usage.

However, I am open to suggestions and would greatly appreciate any insights you might have on how to better handle this. Would you suggest proceeding with the library modification or do you have an alternative solution in mind?